src/briefcase/bootstraps/pursuedpybear.py

```diff
@@ -121,7 +121,7 @@ def pyproject_table_linux_system_arch(self):
 
     def pyproject_table_linux_appimage(self):
         return """
-manylinux = "manylinux2014"
+manylinux = "manylinux_2_28"
```

Comment:
We should flag this change with a `removal` release note (in addition to the feature), as it's a notable change to defaults.

Reply:
I was on the fence about whether this should be a `removal` note since it will only affect new projects. Although, if someone familiar with the old behavior created a wholly new project, they very well could be surprised then.